### PR TITLE
Sell for SFOX using new Payment API

### PR DESCRIPTION
**Pull request by prwelber**

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/scenes/BuySell/SfoxCheckout/Content/template.success.js [hunk 1]
```diff
@@ -38,8 +38,23 @@ const isPending = (t) => t.state === 'processing'
...
+    submitSellQuote,
+    handleTrade,
+    quoteR,
+    sellQuoteR,
+    base,
+    errors,
+    showModal,
+    handleTradeDetailsClick,
+    ...rest } = props
+  console.log('sfox content success:', payment)
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/BuySell/SfoxCheckout/Content/template.success.js]: Small thing but think we should prepend 'buy' to fetchQuote, refreshQuote, quoteR. Makes it more readable

File: packages/blockchain-wallet-v4-frontend/src/data/modules/sfox/sagas.js [hunk 2]
```diff
@@ -100,14 +105,49 @@ export default ({ coreSagas }) => {
...
+      payment = yield payment.amount(parseInt(trade.sendAmount))
+
+      // use priority fee
+      payment = yield payment.fee('priority')
+
+      // assign payment to from trade
+      payment = yield payment.to('153tQjKYMuxRhymRDvoHqcKez94anDGkGF') // TODO this should be "trade.receiveAddress"
+
+      // assign description with id from trade
+      // let trade = { id: 'fakeId12345' }
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/data/modules/sfox/sagas.js]: unnecessary comment?

**Comment by plondon** [comment]
How are errors/loading states handled?

File: packages/blockchain-wallet-v4-frontend/src/scenes/BuySell/SfoxCheckout/Content/template.success.js (hunk 1 above)
Comment (prwelber) [reply packages/blockchain-wallet-v4-frontend/src/scenes/BuySell/SfoxCheckout/Content/template.success.js]: 👍 agree

File: packages/blockchain-wallet-v4-frontend/src/data/modules/sfox/sagas.js (hunk 2 above)
Comment (prwelber) [reply packages/blockchain-wallet-v4-frontend/src/data/modules/sfox/sagas.js]: For sure, this will get cleaned up before merging into main sell branch.

**Pull request merged**